doc/source/examples/03_modeling/sweep_chain_profile.mystnb

```diff
@@ -0,0 +1,134 @@
+---
+jupytext:
+  text_representation:
+    extension: .mystnb
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.1
+kernelspec:
+  display_name: .venv
+  language: python
+  name: python3
+---
+
+# Design: Sweep Chain and Sweep Sketch
+The purpose of this notebook is to demonstrate the `sweep_sketch` and `sweep_chain` functions and their usage for creating more complex extrusion profiles. `sweep_sketch()` is to be used with a closed sketch profile, while `sweep_chain()` is designed for an open profile.
```

Comment:
```suggestion
# Design: Sweep chain and sweep sketch
This notebook shows how use the `sweep_sketch()` and `sweep_chain()` functions to create more complex extrusion profiles. You use the `sweep_sketch()` function with a closed sketch profile and the `sweep_chain()` function for an open profile.
```
Functions or methods? I'm not sure the terms are used consistently/correctly. We should also include the ``()`` in the name, but I'm not sure I caught all earlier occurrences.